# /** ------ 交通标志信息 ------ */
string  au1Type
coord   sPos
uint32  u4TrafficSignId
uint32  u4ObjectId
string  au1SubType
float32 u4RoadDist
float32 u4Value
int8    u1MinLane
int8    u1MaxLane